docs/pages/components/shellbar.md

```diff
@@ -89,62 +89,49 @@ This example includes the product menu for navigating to applications within the
-            </ul>
-          </nav>
-        </div>
+    <div class="fd-popover">
+      <div class="fd-popover__control">
+        <button class="fd-button--light fd-button--menu fd-shellbar__button--menu" aria-controls="9GLB26941" aria-haspopup="true" aria-expanded="false">
+          <span class="fd-shellbar__title">Corporate Portal</span>
+        </button>
+      </div>
+      <div class="fd-popover__body fd-popover__body--no-arrow" aria-hidden="true" id="9GLB26941">
+        <nav class="fd-menu">
+          <ul class="fd-menu__list">
+            <li><a role="button" class="fd-menu__item">Application A</a></li>
+            <li><a role="button" class="fd-menu__item">Application B</a></li>
+            <li><a role="button" class="fd-menu__item">Application C</a></li>
+            <li><a role="button" class="fd-menu__item">Application D</a></li>
+          </ul>
+        </nav>
       </div>
     </div>
-		<div class="fd-shellbar__subtitle">Subtitle</div>
+	<div class="fd-shellbar__subtitle">Subtitle</div>
   </div>
   <div class="fd-shellbar__group fd-shellbar__group--actions">
-      <div class="fd-shellbar__action fd-shellbar__action--desktop">    
-        <div class="fd-search-input">
-         <div class="fd-popover">
-          <div class="fd-popover__control">
-              <div aria-label="Image label" aria-controls="F4GcX348b" aria-expanded="false" aria-haspopup="true">
-                  <div class="fd-input-group">
-                      <input type="text" class="fd-input fd-input-group__input" id="F4GcX348b1" value="Sear" placeholder="Search..." aria-hidden="true">
-                      <span class="fd-input-group__addon fd-input-group__addon--button">
-                          <button class="fd-shellbar__button fd-button sap-icon--search"></button>
-                      </span>
-                  </div>
-              </div>
-          </div>
-          <div class="fd-popover__body fd-popover__body--no-arrow" aria-hidden="true" id="F4GcX348b">
-              <nav class="fd-menu">
-                <ul class="fd-menu__list">
-                    <li><a href="#" class="fd-menu__item"><strong>Sear</strong>ch Result A</a></li>
-                    <li><a href="#" class="fd-menu__item"><strong>Sear</strong>ch Result B</a></li>
-                    <li><a href="#" class="fd-menu__item"><strong>Sear</strong>ch Result C</a></li>
-                    <li><a href="#" class="fd-menu__item"><strong>Sear</strong>ch Result D</a></li>
-                </ul>
-              </nav>
+    <div class="fd-shellbar__action fd-shellbar__action--desktop">    
+      <div class="fd-popover__control">
+        <div aria-label="Image label" aria-controls="F4GcX348b" aria-expanded="false" aria-haspopup="true">
+          <div class="fd-input-group fd-shellbar__input-group">
+            <input type="text" class="fd-input fd-input-group__input fd-shellbar__input-group__input" id="F4GcX348b1" value="Sear" placeholder="Search...">
```

Comment:
Please remove or complete the value from search inputs, it looks weird in the documentation (`value="Sear"`).